products/jbrowse-cli/src/commands/text-index.ts

```diff
@@ -35,61 +35,76 @@ export default class TextIndex extends JBrowseCommand {
     out: flags.string({
       description: 'Synonym for target',
     }),
+
+    attributes: flags.string({
+      description: 'Comma separated list of attributes to index',
+      default: 'Name,ID',
+    }),
```

Comment:
Attributes to index should come from the track config, not the command line. That way, different tracks in an aggregate can have different attributes that are indexed, but can still be indexed all together.

Reply:
Could we possibly use the --attributes flag as an overwrite? If it's not in the command line arguments then it uses the config attributes?